proto/poktroll/migration/event.proto

```diff
@@ -59,3 +59,24 @@ message EventMorseApplicationClaimed {
   // The service ID of the service which the application is staked for.
   string serviceId = 6;
 }
+
+// EventMorseSupplierClaimed is emitted when a MorseAccount is claimed on-chain as a staked Supplier.
+message EventMorseSupplierClaimed {
```

Comment:
I think we should include the staked supplier in this event; this would be consistent with normal supplier staking behavior.